### PR TITLE
Martial arts QOL

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Moves the text describing the combos of your current martial art from the chat to a separate window. Also adds coloring to the intents.

## Why It's Good For The Game
When you don't know the martial art combos by hand you either need to stop the chat from scrolling, which can make it hard to tell if you hand a combo, or constantly spam the button. This allows people to keep the chat scrolling while also keeping an eye out on the list of the moves.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>
Karate moveset window :

image
CQC moveset window : 

image
Sleeping Carp moveset window:

image
Tribal Claw moveset window:

image
Plasma Fist moveset window:

image

</details>

## Changelog
:cl:
add: Color coded the intents in martial art moveset lists.
tweak: Made the "Recall your martial art moves" verbs output their text into a separate window rather than the chat
/:cl:

### CODE REVIEW

File: code/datums/martial/karate.dm [hunk 1]
```diff
@@ -114,10 +114,22 @@
 	set name = "Recall Teachings"
 	set desc = "Remember the martial techniques of Karate."
 	set category = "Karate"
-
+	/*
```
Comment (Penwin0) [inline code/datums/martial/karate.dm]: Don't leave commented code

**Review by Penwin0 — CHANGES_REQUESTED** [review]

File: code/datums/martial/karate.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/datums/martial/karate.dm]: Ah crap, I knew I forgot something.

**Comment by PowerfulBacon** [comment]
Generally, UIs that are being newly added and didn't exist already should be made with TGUI rather than using browse()

**Comment by CydiaLamiales** [comment]
I'm gonna be honest I've got no idea how TGUI works. But if need be I'll remake this PR.

**Review by Archanial — CHANGES_REQUESTED** [review]
This looks good but please make it use TGUI.
We're trying to move away from legacy UIs.

**Pull request closed without merging**